scout/server/blueprints/cases/controllers.py

```diff
@@ -150,6 +151,53 @@ def _populate_case_groups(store, case_obj, case_groups, case_group_label):
             case_group_label[group] = store.case_group_label(group)
 
 
+def _get_partial_causatives(store: MongoAdapter, case_obj: Dict) -> List:
+    """Return any partial causatives a case has, populated as causative objs.
+    Args:
+        store(adapter.MongoAdapter)
+        case_obj(models.Case)
+    Returns:
+        partial(list(dict)): includes the cases, how many there are and the limit.
```

Comment:
What limit? I don't get it?

Reply:
The usual for number for cases to return for a page, as given by user or default, I think. This is old code, and based on the language I'd guess from Måns? I'll add a few more words!

Reply:
Or remove really, that is out of date. Well spotted!